src/languages/en.js

```diff
@@ -70,7 +70,7 @@ export default {
     hello: 'Hello',
     phoneCountryCode: '1',
     welcomeText: {
-        phrase1: 'With Expensify.cash, chat and payments are the same thing.',
+        phrase1: 'Welcome to Expensify.cash! Enter your phone or email to continue.',
```

Comment:
Can you also please ensure that all of the new constants in this file are also added to `es.js`? They can remain in English, we just want to ensure any new keys exist in both files.